RecoBTag/ONNXRuntime/python/pfHiggsInteractionNet_cff.py

```diff
@@ -0,0 +1,31 @@
+import FWCore.ParameterSet.Config as cms
+
+from RecoBTag.FeatureTools.pfDeepBoostedJetTagInfos_cfi import pfDeepBoostedJetTagInfos as _pfHiggsInteractionNetTagInfos
+from RecoBTag.ONNXRuntime.pfDeepBoostedJetTags_cfi import pfDeepBoostedJetTags as _pfHiggsInteractionNetTags
+from RecoBTag.ONNXRuntime.Parameters.HiggsInteractionNet.V00.pfHiggsInteractionNetPreprocessParams_cfi import pfHiggsInteractionNetPreprocessParams
+
+# modify default parameters for tag infos
+pfHiggsInteractionNetTagInfos = _pfHiggsInteractionNetTagInfos.clone(
+    min_pt_for_track_properties = cms.double(0.95),
```

Comment:
please drop type specifications for all parameters which already exist.
This is a safer way to protect from parameter name mistakes and will also help in possible parameter migrations.

```suggestion
    min_pt_for_track_properties = 0.95,
```
same for other parameters cases in this file

Reply:
Done.